internal/services/managementgroup/management_group_resource.go

```diff
@@ -59,6 +59,11 @@ func resourceManagementGroup() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"management_group_id": {
+				Type:     pluginsdk.TypeString,
+				Computed: true,
+			},
```

Comment:
could we call this `tenant_scoped_id`? I'm omitting `management_group` in the field name here since it's in the resource type